packages/playground/website/src/main.tsx

```diff
@@ -10,72 +10,63 @@ import { SiteView } from './components/site-view/site-view';
 import { StorageTypes, StorageType } from './types';
 import { SiteManager } from './components/site-manager';
 import { useBootPlayground } from './lib/use-boot-playground';
-import { Provider } from 'react-redux';
+import { Provider, useSelector } from 'react-redux';
 import { useEffect, useRef, useState } from '@wordpress/element';
-import store from './lib/redux-store';
+import store, { PlaygroundReduxState } from './lib/redux-store';
 import {
 	__experimentalNavigatorProvider as NavigatorProvider,
 	__experimentalNavigatorScreen as NavigatorScreen,
 } from '@wordpress/components';
+import { Blueprint } from '@wp-playground/blueprints';
 
 collectWindowErrors(logger);
 
-const query = new URL(document.location.href).searchParams;
-const blueprint = await resolveBlueprint();
```

Comment:
To support switching between persisted sites, we can no longer just resolve the blueprint once in module scope but rather need to do it within the component every time the selected site changes or the site settings change.

Reply:
Blueprint, as in site setup steps, should only run once when a site is created. To load the site we only need the same runtime configuration, e.g. the PHP version. We might need to rethink this flow entirely, e.g. create a new remote.html iframe instance for each site.